lib/byte_safe_strings.php

```diff
@@ -50,8 +50,10 @@ function RandomCompat_strlen($binary_string)
                     'RandomCompat_strlen() expects a string'
                 );
             }
+
             return mb_strlen($binary_string, '8bit');
         }
+
```

Comment:
why not setup php-cs-fixer rules that can handle most formatting (trailing eol, newline before return can be controlled by it), and contributors can run it before submiting PR